world/quests/test/task-demo.lisp

```diff
@@ -0,0 +1,185 @@
+| (There's human blood on the rock. I can't follow them through there - I need to find another way around.)
+| ! eval (activate 'fi-landslide)
+| ! eval (complete 'fi-check)
+")
+; | ! eval (let (landslide true))
+(quest:interaction :name fi-landslide :interactable fi :dialogue "
+~ player
+| There's been a landslide in the cave.
+~ fi
+| That's not unusual in these parts. But given that everything else seems wrong about this place...
+~ player
+| I hope the android isn't buried under there.
+~ fi
+| Androids are tougher than they look.
+~ player
+| I don't look tough?
+~ fi
+| *Smiles* I didn't say that.
+| [has-more-dialogue Anything more? | Well this is awkward...]")
+(quest:interaction :name fire-check :interactable fire :dialogue "
+~ player
+| (The fire is still warm. They left recently.)
+| ! eval (activate 'fi-fire)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name fi-fire :interactable fi :dialogue "
+~ player
+| I found the remnants of another human camp. Recently vacated.
+~ fi
+| I think it's fair to say that Harris hasn't been completely honest with us.
+~ player
+| You think it's a trap?
+~ fi
+| There's one way to find out.
+~ player
+| Yes... Lucky me.
+~ fi
+| You can handle yourself.
+| [has-more-dialogue Next? | Sayonara.]")
+(quest:interaction :name beacon-check :interactable beacon :dialogue "
+~ player
+| (How did this get all the way up here?)
+| (... Look at that view... Somebody's thinking place?)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name corpse-check :interactable corpse :dialogue "
+~ player
+| (Remains: human. Dismembered, and already picked clean.)
+| (An animal attack? Or those rogue robots, perhaps.)
+| ! eval (complete 'fi-check)
+")
+; \/ query if already know of landslide var, and tweak text e.g. "This is the other side of the landslide"
+(quest:interaction :name cave-in-check :interactable cave-in :dialogue "
+~ player
+| (The cave collapsed here. This is where the humans came through - I can see more bodies underneath.)
+| (The scratches in the rocks suggest combat occurred - swords, gunfire, and finger nails...)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name android-check :interactable android :dialogue "
+~ player
+| (It's the android... It's been destroyed. Not even Catherine could fix this kind of damage.)
+| Hello, sestra... What happened to you?
+| (More human remains. It appears you didn't go down without a fight.)
+| (The memory replay seems to be intact.)
+| Excuse me, sestra... I just need to retrieve this... --- Thank you.
+| (I'd better get this back to Fi.)
+| ! eval (complete 'fi-check)
+| ! eval (activate 'fi-outcome)")
+(quest:interaction :name home-check :interactable home :dialogue "
+~ player
+| (They made a home here. Who was Harris to take that away?)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name home-tent-check :interactable home-tent :dialogue "
+~ player
+| (Androids do use bedrolls. At least, this one did.)
```

Comment:
`Huh, so some androids do sleep after all. And given everything else, I don't think this one did it to blend in, either.`
`Perhaps it brought them comfort?`

Reply:
yeah, I thought this was a nice character detail and callback to the beginning, so a bit more rumination makes sense here 👍 
Dunno if you've seen my level design "skills", but I just copied the tent and put it in the cave :)

Reply:
It looked fine, if a bit stark :)